.env.example

```
@@ -0,0 +1 @@
+REACT_APP_URL_CONTACTS = /contacts
```

Quitar espacios y agregar comillas para anular el margen de error:

```suggestion
REACT_APP_URL_CONTACTS="/contacts"
```